src/controllers/expenseController.js

```diff
@@ -0,0 +1,131 @@
+const expensesService = require('../services/expensesService');
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  try {
+    const filteredExpenses = expensesService.getAllExpenses(
+      userId,
+      categories,
+      from,
+      to,
+    );
+
+    res.status(200).json(filteredExpenses);
+  } catch (error) {
+    res.status(500).json({ error: 'Internal server error' });
+  }
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const expense = expensesService.getExpenseById(id);
+
+    if (!expense) {
+      return res.status(404).json({ error: 'Expense not found' });
+    }
+
+    res.status(200).json(expense);
+  } catch (error) {
+    res.status(500).json({ error: 'Internal server error' });
+  }
```

Comment:
Same issue here with the error handling. It's generally a good idea to have more details about the error, especially in a development environment.